src/applications/mhv/secure-messaging/reducers/breadcrumbs.js

```diff
@@ -1,7 +1,7 @@
 import { Actions } from '../util/actionTypes';
 
 const initialState = {
-  list: [],
+  list: {},

```

Comment:
Was this change intentional?

Reply:
initially, that object was an array. However, there is no need to have an array since we are using only a single level breadcrumb now